coreax/solvers/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""Solvers for generating coresets."""
```

Comment:
Do we need the copyright message here? It is present in every other __init__.

Reply:
Yes, well spotted!